### DISCUSSION TITLE
Windscribe

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#358] [General]
Has anybody used Windscribe? Can it be trusted?

**Comment by hernessteve** [comment]
First time i hear about such a thing. What is that?

**Comment by janply** [comment]
It's VPN. I tried if for my firestick. It is mentioned on [https://www.firesticktricks.com/windscribe-vpn-firestick.html](https://www.firesticktricks.com/windscribe-vpn-firestick.html), it offers military-grade encryption and lots of safe tunneling protocols to keep your data safe. As I know, it's one of the best free VPN services. It has free and paid options, and both possess excellent streaming capabilities.

**Comment by ellielo** [comment]
Thank you guys for all your replies!

**Comment by MirkosMangle** [comment]
Nord vpn is great!
